### PR TITLE
Card component

### PR DESCRIPTION
**Pull request by fadeomar**
Relates #31
**card component :**

----
## UI
cardList
----

***the card component will display in the filter page that allow the user to see the first result we will send it form back-end before he make nay filter process**

  - the component will be a collapse element that show the user the name of therapy and once the user click in the name or in the card in general will open the card that display the therapy information 
 
  - we will use ant design to  make the card 
  - and the propos will contain : 
``` 
{    
    name: 'therpay name',
    priceRange: therapy range price,
    avatar: 'therapy photo',
    approach: 'Therpay approach',
    type: 'Therpay Type',
   }
```

  - run or test the component : 
    - in the file app import the component 
      ``` 
import Card from '../common/Card';
      ```
    - add the component inside the app div
      ```
    <div className="App">
      <Card />
    </div>
      ```

**Fake Data**
   ````
const staticData = [
  {
    name: 'fadi omar',
    priceRange: 9999,
    avatar:
      'http://www.clker.com/cliparts/b/1/f/a/1195445301811339265dagobert83_female_user_icon.svg.med.png',
    approach: 'Therpay approach',
    type: 'Therpay Type',
  },
  {
    name: 'hashem',
    priceRange: 8888,
    avatar: 'https://connectbuncombe.org/wp-content/uploads/2019/05/woman.png',
    approach: 'Therpay approach',
    type: 'Therpay Type',
  },
  {
    name: 'mohmed',
    priceRange: 7777,
    avatar:
      'https://cdn.pixabay.com/photo/2014/04/03/10/07/faceless-309882_960_720.png',
    approach: 'Therpay approach',
    type: 'Therpay Type',
  },
];

export default staticData;
  ````

### CODE REVIEW

File: client/src/components/common/Card/cardContent.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import './style.css';
+
+const CardContent = ({ data: { avatar, name, approach, type } }) => (
+  <div className="Card__content">
```
Comment (MohammadAlhalaq) [inline client/src/components/common/Card/cardContent.js]:
```suggestion
  <div className="card__content">
```

File: client/src/components/common/Card/cardContent.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <img src={avatar} alt={name} className="card__photo" />
+
```
Comment (MohammadAlhalaq) [inline client/src/components/common/Card/cardContent.js]:
```suggestion
```

File: client/src/components/common/Card/staticDataTesting.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+const staticData = [
+  {
+    name: 'fadi omar',
+    priceRange: 9999,
+    avatar:
+      'http://www.clker.com/cliparts/b/1/f/a/1195445301811339265dagobert83_female_user_icon.svg.med.png',
```
Comment (Asmaathabet) [inline client/src/components/common/Card/staticDataTesting.js]: use avatar from ant design as user icon

File: client/src/components/common/Card/index.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+const genExtra = price => (
+  <span className="card__price">{price} $/Session</span>
+);
+const Card = () => {
+  return (
+    <Collapse accordion>
+      {data.map(therpy => (
+        <Panel
+          header={therpy.name}
```
Comment (hashemt) [inline client/src/components/common/Card/index.js]: use destructure

File: client/src/components/common/Card/index.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+import CardContent from './cardContent';
+import data from './staticDataTesting';
+
+const { Panel } = Collapse;
+
+const genExtra = price => (
+  <span className="card__price">{price} $/Session</span>
+);
+const Card = () => {
```
Comment (hashemt) [inline client/src/components/common/Card/index.js]: insert data as props

File: client/src/components/common/Card/style.css [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    align-items: center;
+    color: #242424;
+}
+
+.Card__content {
+    display: flex;
+    justify-content: space-around;
+}
+
+.card__approach, .card__type {
```
Comment (Asmaathabet) [inline client/src/components/common/Card/style.css]:
```suggestion
.card__approach, .card__type {
```
```
.card__approach,
.card__type {
```

Comment (Asmaathabet) [inline client/src/components/common/Card/style.css]:
```suggestion
.card__approach, 
.card__type {
```

File: client/src/components/App/index.js [hunk 7]
```diff
@@ -2,9 +2,14 @@ import React from 'react';
 
 import 'antd/dist/antd.css';
 import './style.css';
+import Card from '../common/Card';
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <Card />
```
Comment (Asmaathabet) [inline client/src/components/App/index.js]: delete this from app

File: client/src/components/common/Card/style.css [hunk 8]
```diff
@@ -0,0 +1,48 @@
+.card__photo {
+    border: 1px solid gray;
+    /* border-radius: 100%; */
```
Comment (MohammadAlhalaq) [inline client/src/components/common/Card/style.css]:
```suggestion
```

File: client/src/components/common/Card/style.css [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+    background-color:#82D9C9; 
+    width:100px;
+}
+
+.card__price {
+    font-family: Roboto;
```
Comment (MohammadAlhalaq) [inline client/src/components/common/Card/style.css]:
```suggestion
    font-family: Roboto, sans-serif;
```

File: client/src/components/App/index.js [hunk 10]
```diff
@@ -2,14 +2,9 @@ import React from 'react';
...
 function App() {
-  return (
-    <div className="App">
-      <Card />
-    </div>
-  );
+  return <div className="App"></div>;
 }
 
 export default App;
```
Comment (Asmaathabet) [inline client/src/components/App/index.js]: insert empty line after this

**Pull request merged**